apps/coinsence-kit/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Application",
+  "description": "An application for Aragon",
+  "icons": [{
+    "src": "/dist/images/icon.png",
```

Comment:
we should remove the the parts that we don't use. 
though I don't even know where all the manifest.json is used.

Reply:
I don't know also where it is used... so will not remove it... just cleaning it